android/src/main/java/com/RNProximityWakeLock/RNReactNativeProximityWakeLockPackage.java

```diff
@@ -16,13 +16,14 @@
       return Arrays.<NativeModule>asList(new RNReactNativeProximityWakeLockModule(reactContext));
     }
 
-    @Override
-    public List<Class<? extends JavaScriptModule>> createJSModules() {
-      return Collections.emptyList();
+    // Deprecated RN 0.47
```

Comment:
I changed this by referencing https://github.com/zmxv/react-native-sound/pull/239/files